SECURITY_CONTACTS

```diff
@@ -0,0 +1,12 @@
+# Reporting a security vulnerability
+
+Please do:
+  - not disclose any security issue publicly e.g. Pull Requests, Issues, Comments.
+  - not disclose any security issue to any owner of the repository or to any other contributor.
```

Comment:
I'd like this line to have word "directly" in it, as now it sounds like we don't want to know at all

Reply:
Dibe